changelog/@unreleased/pr-2541.v2.yml

```diff
@@ -0,0 +1,6 @@
+type: feature
+feature:
+  description: 'Add check: Use InvoationTargetException.getCause instead of getTargetException.
+    Add `isExpressionThis` to MoreASTHelpers.'
```

Comment:
`MoreASTHelpers` is an internal util, not meant as public API -- no need to describe it in changelogs (also, this piece has been reverted ;-)

Reply:
Got it. Also fixed the typo in invocation.